build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
+        classpath 'com.android.tools.build:gradle:4.2.1'
```

Comment:
_This is is unrelated to this pull request._

We cannot change Android Gradle plugin,
as IDEA is not compatible yet.